scripts/install_deps.sh

```diff
@@ -883,101 +964,78 @@ install_deps() {
   environments to ${OUTPUT_ENV_FILE}.\n
-  Please run 'source ${OUTPUT_ENV_FILE}' before run GraphScope."
-  succ ${succ_msg}
-  if [[ ${VERBOSE} == true ]]; then
-    set +x
-  fi
+  Please run 'source ${OUTPUT_ENV_FILE}' before make and run GraphScope."
+  succ "${succ_msg}"
 }
 
 ##########################
-# Main function for build_and_deploy command.
+# Main function for installing dependencies of k8s environment.
 # Globals:
 #   VERBOSE
-#   BUILD_TYPE
-#   INSTALL_PREFIX
 # Arguments:
 #   None
 # Outputs:
 #   output log to stdout, output error to stderr.
 ##########################
-build_and_deploy() {
-
-  # parse args for install_deps command
+install_deps_k8s() {
+  # parse args for install_deps_dev
+  OVERWRITE=false
   while test $# -ne 0; do
     arg=$1; shift
     case ${arg} in
-      --help)        build_and_deploy_usage; exit ;;
-      --verbose)     VERBOSE=true; readonly VERBOSE; ;;
-      --build_type)  BUILD_TYPE=$1; readonly BUILD_TYPE; shift ;;
-      --prefix)
-        if [ $# -eq 0 ]; then
-          echo "there should be given a path for prefix option."
-          build_and_deploy_usage; exit;
-        fi
-        INSTALL_PREFIX=$1; readonly INSTALL_PREFIX; shift ;;
+      --verbose) VERBOSE=true; readonly VERBOSE; ;;
+      --overwrite) OVERWRITE=true; readonly OVERWRITE; ;;
       *)
         echo "unrecognized option '${arg}'"
-        build_and_deploy_usage; exit;;
+        usage; exit;;
     esac
   done
 
   if [[ ${VERBOSE} == true ]]; then
     set -x
   fi
 
-  get_os_version
-
-  check_os_compatibility
-
-  # if .graphscope_env already exists, source it
-  if [ -f "${OUTPUT_ENV_FILE}" ]; then
-    log "Found env file ${OUTPUT_ENV_FILE} exists, source the env file."
-    source ${OUTPUT_ENV_FILE}
+  if [[ ${OVERWRITE} = false && -f "${HOME}/.kube/config" ]]; then
+    warning_msg="We found existing kubernetes config, seems that you already
+    have a ready kubernetes cluster. If you do want to reset the kubernetes
+    environment, please retry with '--overwrite' option."
+    warning "${warning_msg}"
+    exit 0
   fi
 
-  check_dependencies
+  get_os_version
 
-  if [ ${#packages_to_install[@]} -ne 0 ]; then
-    err "The dependences of GraphScope are not satisfied."
-    err "These packages: [${packages_to_install[*]}] are not installed or their version are not compatible."
-    exit 1
-  fi
+  check_os_compatibility_k8s
 
-  install_libgrape-lite
+  install_dependencies_k8s
 
-  install_vineyard
+  start_docker
 
-  install_graphscope
+  launch_k8s_cluster
 
-  succ "GraphScope has been built successfully and installed on ${INSTALL_PREFIX}."
-  succ "Please manually run:"
-  succ "export GRAPHSCOPE_HOME=${INSTALL_PREFIX}"
-  succ "before using GraphScope via Python client, enjoy!"
-  if [[ ${VERBOSE} == true ]]; then
-    set +x
-  fi
+  pull_images
+
+  succ_msg="The script has successfully prepared an environment for GraphScope.
```

Comment:
succ_msg_dev:
"The script has installed all dependencies for building GraphScope, use command \
`$ export XXX`
`$ make graphscope`
to build and develop graphscope."

succ_msg_k8s=
"The script has prepared a local k8s cluster with **kind** to run GraphScope in distributed mode, export essential environment variables with \
`$ export xxxxxx`
before using GraphScope. "